src/cl65/main.c

```diff
@@ -414,6 +415,12 @@ static void SetTargetFiles (void)
     const char* TargetName = GetTargetName (Target);
     unsigned    TargetNameLen = strlen (TargetName);
 
+    if (NoRTL)
+    {
```

Comment:
That option should be implemented in the `Link()` function.  Example:
```c
if (LinkTargetLib) {
    SetTargetFiles ();
    CmdAddArg (&LD65, TargetLib);
}
```

Reply:
I already moved this check to the `Link()` function.